scripts/dev_setup.sh

```diff
@@ -616,6 +618,12 @@ if [[ "$OPERATIONS" == "true" ]]; then
   install_pkg yamllint "$PACKAGE_MANAGER"
   install_pkg python3 "$PACKAGE_MANAGER"
   install_pkg unzip "$PACKAGE_MANAGER"
+  install_pkg jq "$PACKAGE_MANAGER"
+  install_pkg git "$PACKAGE_MANAGER"
+  #for timeout
+  if [[ "$PACKAGE_MANAGER" == "apt-get" ]]; then
+    install_pkg coreutils "$PACKAGE_MANAGER"
```

Comment:
I should delete this line about coretools, already installed, just timeout isn't working.